README.md

```diff
@@ -275,13 +275,13 @@ Returns
+````Python
+ZeroxOutput(completion_time=9432.975, file_name='cs101', input_tokens=36877, output_tokens=515, pages=[Page(content='| Type    | Description                          | Wrapper Class |\n|---------|--------------------------------------|---------------|\n| byte    | 8-bit signed 2s complement integer   | Byte          |\n| short   | 16-bit signed 2s complement integer  | Short         |\n| int     | 32-bit signed 2s complement integer  | Integer       |\n| long    | 64-bit signed 2s complement integer  | Long          |\n| float   | 32-bit IEEE 754 floating point number| Float         |\n| double  | 64-bit floating point number         | Double        |\n| boolean | may be set to true or false         | Boolean       |\n| char    | 16-bit Unicode (UTF-16) character   | Character     |\n\nTable 26.2.: Primitive types in Java\n\n### 26.3.1. Declaration & Assignment\n\nJava is a statically typed language meaning that all variables must be declared before you can use them or refer to them. In addition, when declaring a variable, you must specify both its type and its identifier. For example:\n\n```java\nint numUnits;\ndouble costPerUnit;\nchar firstInitial;\nboolean isStudent;\n```\n\nEach declaration specifies the variable’s type followed by the identifier and ending with a semicolon. The identifier rules are fairly standard: a name can consist of lowercase and uppercase alphabetic characters, numbers, and underscores but may not begin with a numeric character. We adopt the modern camelCasing naming convention for variables in our code. In general, variables must be assigned a value before you can use them in an expression. You do not have to immediately assign a value when you declare them (though it is good practice), but some value must be assigned before they can be used or the compiler will issue an error. \n\nThe assignment operator is a single equal sign, `=` and is a right-to-left assignment. That is, the variable that we wish to assign the value to appears on the left-hand-side while the value (literal, variable or expression) is on the right-hand-side. Using our variables from before, we can assign them values:\n\n> 2 Instance variables, that is variables declared as part of an object do have default values. For objects, the default is `null`, for all numeric types, zero is the default value. For the boolean type, `false` is the default, and the default char value is `\\0`, the null-terminating character (zero in the ASCII table).', content_length=2333, page=1)])
```

Comment:
This section just shows up as a single line in the readme. Is there a better way to print this? 

<img width="871" alt="image" src="https://github.com/user-attachments/assets/8467b68c-27d3-4aba-ae76-4a3ab422c8fe">

Reply:
@tylermaran, let me look into it.

Reply:
Not a big worry. I'll probably take another stab at the readme later this week too. Make sure everything is covered and we have some consistency. 

I'll probably add in a table for the available args on a typescript/python so we have a comparison on feature parity.

Reply:
I checked, but it seems like, wrapping is not possible with code blocks. We can use quote, but in that case we'll lose syntax highlighting. I also asked chatgpt, is response was: 

___
To wrap long single-line Python code in GitHub markdown while preserving syntax highlighting, you can achieve this by breaking the code into multiple lines. However, GitHub markdown doesn't have an automatic text-wrapping feature for code blocks.

Instead, you can manually break the code into smaller chunks where appropriate (such as at operators or after commas) to make it readable. Here's how you can do it:

Before (without wrapping):
```python
print("This is a very long line of Python code that goes on and on without wrapping, and it's hard to read in GitHub markdown.")
```
After (with manual line breaks):
```python
print(
    "This is a very long line of Python code that goes on and on "
    "without wrapping, and it's hard to read in GitHub markdown."
)
```
By breaking it up into multiple lines, the syntax highlighting will still work, and the readability will improve.

If you can't modify the code directly, you can scroll horizontally in GitHub markdown to see the entire line, but manual wrapping is the best solution for readability.

___
Let me know how you want to proceed.

Reply:
@tylermaran, checkout the readme updated in the latest commit, I followed the manual wrapping way as in the js output part of the readme. Now seems to look much better.